src/main/java/uk/org/tombolo/importer/phe/AdultObesityImporter.java

```diff
@@ -61,65 +57,165 @@ public DatasourceSpec getDatasourceSpec(String datasourceIdString) throws Except
+            }
+            url = file.toURI().toURL();
+        }
+
+        InputStreamReader isr = new InputStreamReader(
+                downloadUtils.fetchInputStream(url, getProvider().getLabel(), ".csv"));
+
+        // Parsing our csv file
+        CSVParser csvFileParser = new CSVParser(isr, format);
+        csvRecords = csvFileParser.getRecords();
+
+
+        // We discard the first 6 records in our data file as these don't hold any meaningfull information.
+        // We do this  calling an iterator object and simply ignoring them:
+        Iterator<CSVRecord> rowIterator = csvRecords.iterator();
+
+        // skipping first 6 rows
+        rowIterator.next();
+        rowIterator.next();
+        rowIterator.next();
+        rowIterator.next();
+        rowIterator.next();
+        rowIterator.next();
+
+        // Looping through the rows of the .csv file
+        while (rowIterator.hasNext()){
+            CSVRecord row = rowIterator.next();
+            try{
+                Subject subject = SubjectUtils.getSubjectByTypeAndNameUnique(localauthority, String.valueOf(row.get(2)).trim());
+
+                // Checking not matched geometries
+                if (subject!=null){
+                    // Creating the time index
+                    for (int timeValuesIndex = 3; timeValuesIndex <= 5; timeValuesIndex++) {
+
+                        CSVRecord rowTime = (CSVRecord) csvRecords.get(5);
+                        String year = rowTime.get(timeValuesIndex).toString();
+
+                        // Cleaning the year record as it appears as: 2014 (Mid-January 2014 to Mid-January 2015)
+                        year = year.substring(0, 4);
+                        LocalDateTime timestamp = TimedValueUtils.parseTimestampString(year);
+
+                        // The value is a string in our .csv file. We need to clean it before using it.
+                        // We  need to check for invalid rows so we will suround this with a try catch clause
+                        try {
+                            String recordString = row.get(timeValuesIndex).replace("%", "");
+                            String attributeName = row.get(0);
+                            try {
+
+                                Double record = Double.parseDouble(recordString);
+                                // Here is where we are assigning the values of our .csv file to the attribute fields we
+                                // created.
+                                if (attributeName.contains("Underweight")){
+                                    Attribute attribute = datasource.getTimedValueAttributes().get(0);
+                                    timedValues.add(new TimedValue(
+                                            subject,
+                                            attribute,
+                                            timestamp,
+                                            record/100.));
+                                }
```

Comment:
why the record is getting divided by 100?

Reply:
The dataset contains percentages however teh attributes mention fractions in the original importer. Converted them to fractions to be consistent

Reply:
sorry, i didn't know that, i thought its one pr